tex/main/MarkovChain/closed_form_state_probs/main.tex

```diff
@@ -0,0 +1,662 @@
+For instance consider the coefficients \((p_1,p_2,p_3)\) of some of the terms from the equations above:
+
+\begin{multicols}{2}
+    \begin{itemize}
+        \item (\ref{eq:00_rate_1131}) \( \Rightarrow (\lambda^A)^2\): \(G(2,0,0) = \binom{2+0}{0} = 1\)
+        \item (\ref{eq:00_rate_1131}) \( \Rightarrow \lambda^A \lambda^o\): \(G(1,1,0) = \binom{1+1-1}{1} = 1\)
+        \item (\ref{eq:00_rate_1131}) \( \Rightarrow 2 \lambda^A \mu\): \(G(1,0,1) = \binom{1+1}{1} = 2\)
+        \item (\ref{eq:00_rate_1131}) \( \Rightarrow \mu^2\): \(G(0,0,2) = 1\)
+        \item (\ref{eq:00_rate_1141}) \( \Rightarrow 2(\lambda^A)^2 \lambda^o\): \(G(2,1,0) = \binom{2+1-1}{1} = 2\)
+        \item (\ref{eq:00_rate_1141}) \( \Rightarrow 3(\lambda^A)^2 \mu\): \(G(2,0,1) = \binom{2+1}{1} = 3\)
+        \item (\ref{eq:00_rate_1141}) \( \Rightarrow 3 \lambda^A \mu^2\): \(G(1,0,2) = \binom{1+2}{2} = 3\)
+        \item (\ref{eq:00_rate_1151}) \( \Rightarrow 3 (\lambda^A)^3 \lambda^o\): \(G(3,1,0) = \binom{3+1-1}{1} = 3\)
+        \item (\ref{eq:00_rate_1151}) \( \Rightarrow 3 (\lambda^A)^2 (\lambda^o)^2 \): \(G(2,2,0) = \binom{3}{2} = 3\)
+        \item (\ref{eq:00_rate_1151}) \( \Rightarrow 6 (\lambda^A)^2 \mu ^ 2\): \(G(2,0,2) = \binom{2+2}{2} = 6\)
+    \end{itemize}
+\end{multicols}
+
+\begin{itemize}
+    \item (\ref{eq:00_rate_1151}) \( \Rightarrow 6 (\lambda^A)^2 \lambda^o \mu\): \(G(2,1,1) = \binom{2+1+1}{2} + 1\binom{2+1}{1+1} - \binom{2+1}{1} = 6 + 3 - 3 = 6\)
+    \item \small{(e.g)} \( \Rightarrow (\lambda^A)^2 (\lambda^o)^2 \mu\): \(G(2,2,1) = \binom{2+2+1}{2} - \binom{2+2-1}{2-1} + \sum_{i=2}^{2+2-2} i\binom{i-1}{2-1} = 10 - 3 + (2 \times 1) = 9\)
+\end{itemize}
+
+
+
+\subsubsection{Unknown terms}
+
+The terms that remain unknown are the terms where \(p_1, p_2, p_3 \geq 2\). 
+For small models these terms are very few but as the model grows the number of these terms increase exponentially. 
+Here are some of these values with the corresponding values of the \(G(p_1,p_2,p_3)\) function.
+
+\begin{multicols}{2}
+    \begin{itemize}
+        \item \(G(2,2,2) = 18\) 
+        \item \(G(3,2,2) = 60\)
+        \item \(G(2,3,2) = 24\)
+        \item \(G(2,2,3) = 30\)
+        \item \(G(4,2,2) = 150\)
+        \item \(G(3,3,2) = 100\)
+        \item \(G(3,2,3) = 120\)
+        \item \(G(2,4,2) = 30\)
+        \item \(G(2,3,3) = 40\)
+        \item \(G(2,2,4) = 45\)
+    \end{itemize}
+\end{multicols}
+
+\subsubsection{Transforming the spanning tree problem into a permutation problem}
+
+Given the nature of the spanning tree problem and the properties that the specific Markov chain model holds, this problem can be further reduced to another problem that may be slightly easier to solve. 
+Consider the following Markov model and the spanning trees rooted at state \((0,0)\) that are associated with it. 
```

Comment:
```suggestion
In this section a new combinatorial object is defined: LDR arrays. It will be shown that there is a 
bijection between LDR arrays and the ... . LDR arrays will then be enumerated which in 
turn enumerates the ....
```